packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyState.twig

```diff
@@ -28,56 +26,60 @@
     {{ styleProp(_styleProps) }}
     {{ classProp(_classNames) }}
 >
-    <div class="{{ _wrapperClassName }}">
-        {% if _iconName %}
-            <div class="{{ _iconWrapperClassName }}">
-                <Icon name="{{ _iconName }}"/>
-            </div>
-        {% endif %}
-        <div>
-            <Heading
-                elementType="{{ _headingElementType }}"
-                size="xsmall"
-                data-test-empty-tile="{{ _headingElementType }}"
-                UNSTABLE_className="{{ _headingClassName }}"
-            >
-                {{ _title }}
-            </Heading>
-            <Text color="secondary">{{ _description | raw }}</Text>
-        </div>
-    </div>
-    {% if block('content') is not empty %}
+    {% if _heading or _description or _imageSlot %}
```

Comment:
You can use `{% if myVar in someOtherArray|keys %}` and some cycle (each) to set a new variable that will hold the information about this condition.

Reply:
refactored (different solution)